buildSrc/src/main/groovy/org.apache.fineract.dependencies.gradle

```diff
@@ -68,7 +68,7 @@ dependencyManagement {
             exclude 'org.codehaus.groovy:groovy'
         }
         dependency 'org.apache.commons:commons-csv:1.10.0'
-        dependency 'org.quartz-scheduler:quartz:2.3.2'
+        dependency 'org.quartz-scheduler:quartz:2.5.0-rc1'
```

Comment:
i dont think we should use RC version. I am not convinced the CVE is effecting us. I am still digesting it.

Reply:
About Quartz library I can see that they have not released any security fix and project seems to be abandoned

https://github.com/quartz-scheduler/quartz/issues/1134

Anyway the last version is 2.5.0-rc1, so then the question is; 

Should we use a Release Candidate version? If not should we keep using a library version released in 2019 ?

Reply:
https://github.com/quartz-scheduler/quartz/issues/943

We dont have quartz-job artifact, neither the class thats the CVE is talking about.

I dont think the reported CVE applies to us... 

You might wanna discuss internally what steps we should do.
@vidakovic Any thoughts?

Reply:
Comments about the issue https://github.com/quartz-scheduler/quartz/issues/943 for now I have rollbacked to the original version.

Reply:
@jdailey @vidakovic @adamsaghy I think this should be a false positive. Anyway the Quartz library maintenance/very low activity is a red flag, becaue the library now seems abandoned.